src/components/InstructorPage.js

```diff
@@ -0,0 +1,257 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import { withRouter } from 'react-router-dom'
+
+import Button from '@material-ui/core/Button'
+
+import './InstructorPage.css'
+
+//Services
+import peerReviewService from '../services/peerReview'
+
+//Actions
+//import instructorPageActions from '../reducers/actions/instructorPageActions'
+
+const GroupDetails = ({ myGroup }) => {
+  if (!myGroup) {
+    return (
+      <div>
+        <h2>This user is currently not part of any group.</h2>
+      </div>
+    )
+  } else {
+    return (
+      <div>
+        {myGroup.map((member, index) => {
+          return (
+            <p key={index}>
+              {index + 1}. {member}
+            </p>
+          )
+        })}
+      </div>
+    )
+  }
+}
+
+const Answers = ({ answers }) => {
+  if (answers) {
+    return (
+      <div>
+        {answers.map((projectGroup, index) => {
+          return (
+            <div key={index}>
+              <hr />
+              <br />
+              <h1>{projectGroup.group.name}</h1>
+              <h3>Ohjaaja: {projectGroup.group.instructorName}</h3>
```

Comment:
The whole app is english, so perhaps all in this page should be also english. Now some details are in english and others in finnish.

Reply:
Questions in the set are all in Finnish so if the text is in English, there's going to be both in the same page. I changed it to English now anyway to be consistent.